packages/nimble-components/src/rich-text-mention/users/tests/rich-text-mention-users.spec.ts

```diff
@@ -0,0 +1,316 @@
+import { html, repeat } from '@microsoft/fast-element';
+import { RichTextMentionUsers, richTextMentionUsersTag } from '..';
+import type { MappingUserKey } from '../../../mapping/base/types';
+import { MappingUser, mappingUserTag } from '../../../mapping/user';
+import { mappingTextTag } from '../../../mapping/text';
+import { Fixture, fixture } from '../../../utilities/tests/fixture';
+import { iconAtTag } from '../../../icons/at';
+import { waitForUpdatesAsync } from '../../../testing/async-helpers';
+
+interface BasicUserMentionMapping {
+    key?: MappingUserKey;
+    displayName?: string;
+}
+
+async function setUserMappingElements(
+    element: RichTextMentionUsers,
+    mappings: BasicUserMentionMapping[]
+): Promise<void> {
+    const newUserMappingElements: MappingUser[] = [];
+    mappings.forEach(mapping => {
+        const mappingUser = document.createElement(
+            mappingUserTag
+        ) as MappingUser;
+        mappingUser.key = mapping.key ?? '';
+        mappingUser.displayName = mapping.displayName ?? '';
+        newUserMappingElements.push(mappingUser);
+    });
+    element.replaceChildren(...newUserMappingElements);
+    await waitForUpdatesAsync();
+}
+
+describe('RichTextMentionUsers', () => {
+    let element: RichTextMentionUsers;
+    let connect: () => Promise<void>;
+    let disconnect: () => Promise<void>;
+
+    // prettier-ignore
+    async function setup(
+        mappings: BasicUserMentionMapping[],
+        pattern = ''
+    ): Promise<Fixture<RichTextMentionUsers>> {
```

Comment:
Can you do the same type of pattern I did here for changing to a configuration object: https://github.com/ni/nimble/blob/main/packages/nimble-components/src/table-column/icon/tests/table-column-icon.spec.ts#L51-L55

It makes it easier to see the exact configuration change between tests

Reply:
Updated as per your suggestion